lib/facets/create-people-facets.js

```diff
@@ -0,0 +1,22 @@
+module.exports = function createPeopleFacets (results) {
```

Comment:
This can be _re-written_ as:
```js
module.exports = function createPeopleFacets (results) {
  var p = results.aggregations.people;
  return { // need brackets around the returned object 
    occupation: p.occupation.occupation_filters.buckets.map(v => ({ value: v.key, count: v.doc_count })),
    place_born: p.place_born.place_born_filters.buckets.map(v => ({ value: v.key, count: v.doc_count })),
    organisation: p.organisation.organisations_filters.buckets.map(v => ({ value: v.key, count: v.doc_count }))
  };
};
```